corehq/apps/user_importer/validation.py

```diff
@@ -49,6 +49,24 @@ def get_user_import_validators(domain_obj, all_specs, allowed_groups=None, allow
     ]
 
 
+# TODO: confirm validators and incorporate into validation check for web users
+def get_web_user_import_validators(domain_obj, all_specs, allowed_roles=None, upload_domain=None):
+    domain = domain_obj.name
+    return [
+        UsernameTypeValidator(domain),
+        #UsernameValidator(domain),
```

Comment:
This can probably be shared with mobile user imports (perhaps with a way to mark if a validator is mobile only). That way if future validations are added, they do not need to be added in two places.